src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -0,0 +1,284 @@
+            # required parameters
+            self.assertEqual(device['cbsdCategory'], cbsd_record[0]['cbsdCategory'])
+            self.assertEqual(device['fccId'], cbsd_record[0]['fccId'])
+            self.assertEqual(device['airInterface']['radioTechnology'],\
+                             cbsd_record[0]['airInterface']['radioTechnology'])
+            self.assertEqual(device['installationParam']['latitude'],\
+                             cbsd_record[0]['installationParam']['latitude'])
+            self.assertEqual(device['installationParam']['longitude'],\
+                             cbsd_record[0]['installationParam']['longitude'])
+            self.assertEqual(device['installationParam']['height'],\
+                             cbsd_record[0]['installationParam']['height'])
+            self.assertEqual(device['installationParam']['heightType'], \
+                             cbsd_record[0]['installationParam']['heightType'])
+            self.assertEqual(device['installationParam']['antennaGain'], \
+                             cbsd_record[0]['installationParam']['antennaGain'])
+             # parameters should exist in record if exist in device,\
+             # if exist only in record, its value should be equal to a default value          
+            self.assertEqualToDeviceParamOrIfExistWithDefault('indoorDeployment', \
+                device['installationParam'], cbsd_record[0]['installationParam'], False)
+            
+            self.assertEqualToDeviceParamOrIfExistWithDefault('antennaAzimuth', \
+                device['installationParam'], cbsd_record[0]['installationParam'], 0)
+            
+            self.assertEqualToDeviceParamOrIfExistWithDefault('antennaDowntilt', \
+                device['installationParam'], cbsd_record[0]['installationParam'], 0)
+            
+            self.assertEqualToDeviceParamOrIfExistWithDefault('antennaBeamwidth', \
+                device['installationParam'], cbsd_record[0]['installationParam'], 0)
+            # if callSign exist, it should have the same value as registered
+            if 'callSign' in cbsd_record[0]:
+                self.assertEqual(device['callSign'], cbsd_record[0]['callSign'])
+            max_eirp_by_MHz = 37;           
+            # if eirpCapacity in the record, it should be the same as device or in accepted limits
+            if 'eirpCapability' in cbsd_record[0]:
+                if 'eirpCapability' in device:
+                    self.assertEqual(device['eirpCapability'], cbsd_record[0]['eirpCapability'])
+                    max_eirp_by_MHz = cbsd_record[0]['eirpCapability'] - 10
+                else:
+                    self.assertLessEqual(cbsd_record[0]['eirpCapability'], 47)
+                    self.assertGreaterEqual(cbsd_record[0]['eirpCapability'], -127)
+            # antennaModel if exists in device should exist with same value in record              
+            if 'antennaModel' in device['installationParam'] \
```

Comment:
I think you can only keep the second half of this 'if' statement: otherwise we would not catch the case where the CBSD did not provide antennaModel but for some reason the SAS did.

I actually typed a comment suggesting you make the callSign check look more like this 'if' statement, then realized the problem I just described. If you remove the first half and `device` *doesn't* have antennaModel, you will get an error on line 146 anyway (it's a little messy to debug but it should be an uncommon error and the code is much cleaner this way).

Edit: I just read the 'else' part of this statement, which I think prevents the condition I was worried about. The logic's a little confusing but I think it's correct.